plugin.rb

```diff
@@ -81,6 +82,31 @@ class Engine < ::Rails::Engine
       end
     end
   end
+
+  on(:post_created) do |post|
+    name = DiscourseAutomation::Triggerable::POST_CREATED_EDITED
+
+    DiscourseAutomation::Trigger.where(name: name).find_each do |trigger|
+      trigger.run!(
+        'kind' => DiscourseAutomation::Triggerable::POST_CREATED_EDITED,
+        'action' => :create,
+        'post' => post
+      )
+    end
+  end
+
+  on(:post_edited) do |post|
+    name = DiscourseAutomation::Triggerable::POST_CREATED_EDITED
+
+    DiscourseAutomation::Trigger.where(name: name).find_each do |trigger|
+      trigger.run!(
+        'kind' => DiscourseAutomation::Triggerable::POST_CREATED_EDITED,
+        'action' => :edit,
+        'post' => post
+      )
+    end
+  end
+    script do
```

Comment:
Is this line supposed to be here?

Reply:
no indeed thanks some error while choosing lines of commit apparently